### PR TITLE
feat: customizing app with dream11 configuration

### PR DESCRIPTION
**Pull request by AyaMahmoud148**
## Description of the change

> Customize the hybrid App with Dream11 Customization 
   1- integrating with codepush
   2- customizing networking for android and ios for more details [check DOCS](https://instabug.atlassian.net/wiki/spaces/CP/pages/4574838874/Dream11+Hybrid+Setup)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> [Issue links go here](https://instabug.atlassian.net/browse/MOB-16272)

### Development

- [x] Lint rules pass locally
- [X] The code changed/added as part of this pull request has been covered with tests

### Code review

- [X] This pull request has a descriptive title and information useful to a reviewer
- [X] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: examples/hybrid/ios/HybridSampleApp/AppDelegate.m [hunk 1]
```diff
@@ -2,10 +2,11 @@
 #import <RNInstabug/RNInstabug.h>
 @implementation AppDelegate
 
-static NSString *const appToken = @"deb1910a7342814af4e4c9210c786f35";
+static NSString *const appToken = @"0fcc87b8bf731164828cc411eccc802a";
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    [RNInstabug initWithToken:appToken invocationEvents:IBGInvocationEventFloatingButton];
+    [Instabug setCodePushVersion:@""];
```
Comment (a7medev) [inline examples/hybrid/ios/HybridSampleApp/AppDelegate.m:8]: Same as Android. We can remove this.

**Review by a7medev — CHANGES_REQUESTED** [review]

**Review by a7medev — APPROVED** [review]
Great job @AyaMahmoud148! 🚀

File: examples/hybrid/App.tsx [hunk 2]
```diff
@@ -23,13 +26,47 @@ const throwUnhandled = () => {
...
+  } catch (error) {
+    console.error('Error:', error);
+  }
+};
+
 const App: () => ReactNode = () => {
   useEffect(() => {
     Instabug.init({
-      token: 'deb1910a7342814af4e4c9210c786f35',
+      token: '0fcc87b8bf731164828cc411eccc802a',
```
Comment (ahmedAlaaInstabug) [inline examples/hybrid/App.tsx:59]: Which project is this token for?

**Review by ahmedAlaaInstabug — APPROVED** [review]

File: examples/hybrid/App.tsx (hunk 2 above)
Comment (AyaMahmoud148) [reply examples/hybrid/App.tsx:59]: React Native beta app https://dashboard.instabug.com/applications/reactnative/beta/settings/sdk-integrations

**Pull request merged**